packages/ui/src/components/va-select/components/VaSelectOptionList/VaSelectOptionList.vue

```diff
@@ -210,18 +225,35 @@ export default defineComponent({
       return searchBaseOrdered.slice(startIndex).find((option) => !getDisabled(option))
     }
 
+    const previousOptionComputed = computed((): SelectOption | undefined => {
+      const previousOptionIndex = currentOptionIndex.value - 1
+      const previousOption = currentOptions.value[previousOptionIndex]
+      const previousOptionCheck = isValueExists(previousOption) && !(previousOptionIndex === 0 && getDisabled(previousOption))
+
+      if (previousOptionCheck) {
+        return findNextActiveOption(currentOptionIndex.value - 1, true)
+      }
+
+      return undefined
```

Comment:
Redundant `undefined`.

Reply:
It's not redundant -> eslint rules forces computed to have obvious return value.

Issue with `0` not related to this pr, will move it separately.

Reply:
How about `null`. It is not undefined, it is empty.

Reply:
It's not empty, it doesn't exist, undefined. Moreover, `findNextActiveOption` may return `undefined` if `find` method will find nothing. I don't think it's a good case to have one computed to return both `null` and `undefined` in same situations.